src/controllers/softtakeover.cpp

```diff
@@ -104,29 +103,29 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
      *          (regardless of the new value's arrival time)
      *      - the new value arrives awhile after the previous one
      *          (regardless of what the previous value was)
-     * 
+     *
      * Sheesh, this is much easier to show in a truth table!
-     * 
+     *
      * Sides    prev distance   new distance    new value arrives   Ignore
      * opposite close           far             later               TRUE
      * opposite far             far             later               TRUE
      * same     close           far             later               TRUE
      * same     far             far             soon                TRUE
      * same     far             far             later               TRUE
-     * 
+     *
      *      Don't ignore in every other case.
      */
 
-    qint64 currentTime = Time::elapsedMsecs();
+    mixxx::Duration currentTime = Time::elapsed();
     // We will get a sudden jump if we don't ignore the first value.
-    if (m_time == 0) {
+    if (m_time == mixxx::Duration::fromMillis(0)) {
```

Comment:
Hm, I'm curious what you mean by "more explicit"? I think it's more explicit to use the factory methods -- in this case the SoftTakeover logic is all written in terms of milliseconds so the unit here matches.

I only created the no-args constructor so that it can be used in containers and not explicitly initialized in class member variables.